web/fotorama/fotorama.js

```diff
@@ -3531,6 +3532,27 @@ fotoramaVersion = '4.6.4';
             }
         }
 
+        /**
+         * #86971
+         * clicking close fullscreen icon on mobile causes minicart opening
+         * this is known fotorama issue:
+         * https://github.com/artpolikarpov/fotorama/issues/530
+         * this is temporary fix
+         */
+        function preventCloseFullscreenEventPropagation() {
+            const cancelCartMouseDown = (e) => {
```

Comment:
not sure if it's a good idea to use arrow functions in a file that isn't `.babel`